plugin/metrics/prometheus/options.go

```diff
@@ -75,6 +113,19 @@ func (opt *Options) InitFromViper(v *viper.Viper) error {
 	cfg.ServerURL = stripWhiteSpace(v.GetString(cfg.namespace + suffixServerURL))
 	cfg.ConnectTimeout = v.GetDuration(cfg.namespace + suffixConnectTimeout)
 	cfg.TokenFilePath = v.GetString(cfg.namespace + suffixTokenFilePath)
+	cfg.MetricNamespace = v.GetString(cfg.namespace + suffixMetricNamespace)
+	cfg.CallsMetricName = v.GetString(cfg.namespace + suffixCallsMetricName)
+	cfg.LatencyMetricName = v.GetString(cfg.namespace + suffixLatencyMetricName)
+	cfg.LatencyUnit = v.GetString(cfg.namespace + suffixLatencyUnit)
+	cfg.OperationLabel = v.GetString(cfg.namespace + suffixOperationLabel)
+
+	if v.IsSet(cfg.namespace + suffixLatencyUnit) {
```

Comment:
I'm not sure if this is the best place to perform config validation; let me know if it should be done somewhere else.